include/private/tdyoptions.h

```diff
@@ -0,0 +1,24 @@
+#if !defined(TDYOPTIONS_H)
+#define TDYOPTIONS_H
+
+#include <petsc.h>
+
+typedef struct{
+    PetscReal gravity_constant;
+
+    // Default values for material properties
+    PetscReal default_porosity;
```

Comment:
These aren't really "defaults", are they? It looks like they are initialized to a default and then possibly overridden by command line arguments. If I've understood this properly, maybe we should just get rid of the `default_` prefix.

Reply:
(Looking below) or maybe we mean "default" in the sense of "numeric value" instead of a function that computes it? In that case the `default_` prefix makes more sense.

Reply:
> mean "default" in the sense of "numeric value" instead of a function

Yes, that was my intention.